package.json

```
@@ -19,7 +19,7 @@
   "main": "./dist/index.js",
   "types": "./dist",
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=12.0.0"
```

```suggestion
    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
```

As per the package.json of eslint